app/components/captureTasksController.js

```diff
@@ -0,0 +1,24 @@
+app.controller('CaptureTasksController', [function(){
+  var self = this
+  this.tasks = []
+  this.taskFilter = {}
```

Comment:
Maybe bring this up with Dan next time you see him...what are his thoughts on using an empty object in this way?